.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,44 @@
+name: Bug Report
+description: File a bug/issue
+title: "[BUG] <title>"
+labels: ["bug", "needs-triage"]
```

We use a lot of prow-generated labels.

```suggestion
labels: ["kind/bug", "needs-triage"]
```

---

Updated in current push